Add an error diag to the converter from static to flow when `scrape_i… (#6270)
grafana/agent: CHANGELOG.md

- Fix an issue where blocks having the same type and the same label across
  modules could result in missed updates. (@thampiotr)

> - Add an error diag to the converter from static to flow when `scrape_integration` is set
>   to true but no remote_write is defined. (@erikbaranowski)

Reviewer: ```suggestion
- An error will be returned in the converter from Static to Flow when `scrape_integration` is set
  to `true` but no `remote_write` is defined. (@erikbaranowski)
```
I think this is a bit more clear.
